packages/core/admin/admin/src/content-manager/components/BlocksInput/utils/conversions.ts

```diff
@@ -0,0 +1,54 @@
+import { type Element, Editor, Transforms } from 'slate';
+
+const prepareHandleConvert = (editor: Editor) => {
```

Comment:
Can we add some documentation here, I think the name is probably appropriate to what you're doing, but being someone who hasn't worked with Slate, it's ambiguous 👍🏼

Reply:
very clear 👍🏼